src/Costumes.cpp

```diff
@@ -54,7 +54,7 @@ struct PlayerState
     {
         if (morph)
         {
-            delete morph;
+            // delete morph;
```

Comment:
Let's fix this before merging 😛

Reply:
I didn't realize how to fix this :(

Reply:
@Helias can you see if the warning is gone now? I don't get this one with my Visual Studio I think